cmd/lakectl/cmd/common_helpers.go

```diff
@@ -267,3 +270,43 @@ func MustParsePathURI(name, s string) *uri.URI {
 	}
 	return u
 }
+
+func IsValidAccessKeyID(accessKeyID string) bool {
+	return (len(accessKeyID) == 20 && (strings.HasPrefix(accessKeyID, "ASIA") || strings.HasPrefix(accessKeyID, "AKIA")) && (string(accessKeyID[4]) == "I" || string(accessKeyID[4]) == "J") && (strings.HasSuffix(accessKeyID, "A") || strings.HasSuffix(accessKeyID, "Q")) && IsContainOnlyUpperLettersOrNumber(accessKeyID))
+}
+
+func IsValidSecretAccessKey(secretAcceessKey string) bool {
+	return (IsBase64(secretAcceessKey) && len(secretAcceessKey) == 40)
+}
+
+func IsBase64(s string) bool {
+	_, err := base64.StdEncoding.DecodeString(s)
+	return err == nil
+}
+
+func IsContainOnlyUpperLettersOrNumber(s string) bool {
+	for _, r := range s {
```

Comment:
I think a simple regexp check will save the need to iterate over the string and will avoid the problem @arielshaqed has pointed in the next line.
Something like:
```
var uppersAndDigitsRegexp = regexp.MustCompile(`\b[0-9A-Z]+\b`)

func IsContainOnlyUpperLettersOrNumber(s string) bool {
	return uppersAndDigitsRegexp.MatchString(s)
}
```
WDYT?

Reply:
changed 👍